doc/core/grpc-error.md

```diff
@@ -71,24 +71,24 @@ errors, or pass them along to a function that would take over the ownership.
 A `grpc_closure` callback function is any function that has the signature:
 
 ```C
-void (*cb)(grpc_exec_ctx *exec_ctx, void *arg, grpc_error *error);
+void (*cb)(void *arg, grpc_error *error);
 ```
 
 This means that the error ownership is NOT transferred when a functions calls:
 
 ```C
-c->cb(exec_ctx, c->cb_arg, err);
+c->cb(c->cb_arg, err);
 ```
 
 The caller is still responsible for unref-ing the error.
 
-However, the above line is currently being phased out! It is safer to invoke
-callbacks with `GRPC_CLOSURE_RUN` and `GRPC_CLOSURE_SCHED`. These functions are
-not callbacks, so they will take ownership of the error passed to them.
+Note that you'll likely never need to run `c->cb(...)` yourself; the idiomatic
+way to execute callbacks is via the `ExecCtx::Run` method, which takes ownership
```

Comment:
we would ideally want Closure::Run() instead of ExecCtx::Run() if possible. ExecCtx::Run() should only be used if needed.

Reply:
Corrected, thanks.